controllers/supervisor/supervisor.py

```diff
@@ -207,6 +208,13 @@ def __init__(self):
         for t in range(constants.NUM_COMMENTS):  # fill with dummies
             self.comments_.append('')
 
+    def step(self, timeStep):
```

Comment:
Time does not stop when the game is in pause state (such as WAIT_GOAL_MS).

Reply:
OK, in such a case, I believe we would need to add a boolean argument to the step function, that will not advance the game time in case of WAIT_GOAL_MS and other waiting times. If you agree, I will implement this.

Reply:
OK